src/style.css

```diff
@@ -0,0 +1,23 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background: url(images/star.svg) no-repeat center;
+  background-position: 50%;
+  margin-right: 4px;
```

Comment:
The 'background-position' property should be combined with the 'background' property on line 12.